src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,5 +1,11 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+
+import java.util.ArrayList;
 import java.util.List;
 
 public interface MachineProducer {
```

Comment:
According to the checklist, you should not use raw types. Always parametrize interfaces and classes if it is possible. In this case, the MachineProducer interface should be parameterized so that it can only produce Machine objects or its subtypes.